generated-platform-project/quarkus-universe/properties/pom.xml

```diff
@@ -13,7 +13,7 @@
   <packaging>pom</packaging>
   <name>Quarkus Platform - Quarkus Universe - Quarkus Platform Properties</name>
   <properties>
-    <platform.quarkus.native.builder-image>graalvm</platform.quarkus.native.builder-image>
+    <platform.quarkus.native.builder-image>mandrel</platform.quarkus.native.builder-image>
```

Comment:
@gsmet should this not stay graalvm?

Reply:
No we had to switch to use Mandrel by default because of a nasty bug in GraalVM that is not fixed yet and for which we don't have any ETA.